src/tag/scss/mixin.scss

```diff
@@ -20,7 +20,7 @@
     line-height: $height - $borderWidth * 2;
     font-size: 0;
 
-    $_marginLeft: $height / 4 - 1px;
+    $_marginLeft: $height / 4 + 3;
```

Comment:
you mean `+3px` ?
@tao1991123 I'm not sure is that ok adding `3px` to marginLeft. Looks like a hack

Reply:
what if we did ` $_marginLeft: $iconSize`? This seems to have the desired result without a fixed hack

Reply:
![image](https://user-images.githubusercontent.com/10049465/54251446-3f34ca00-4582-11e9-90ff-07069e90c5c2.png)
it seem that marginLeft equals to 0.5 * paddingRight

Reply:
agree with `$_marginLeft: $iconSize`